clients/admin-ui/src/pages/settings/domain-records.tsx

```diff
@@ -1,114 +0,0 @@
-/* eslint-disable react/no-unstable-nested-components */
```

Comment:
I can revert it. I didn't see it linked from anywhere so I figured it wasn't being used. I'll add a note that it's a direct access page instead.

Reply:
It looks like it's being used and it can be accessed through the menu but it's a menu option that requires to be in Fides Cloud.
`        title: "Domain verification",
        path: routes.DOMAIN_RECORDS_ROUTE,
`